lighthouse-cli/printer.ts

```diff
@@ -110,7 +110,13 @@ function createOutput(results: Results, outputMode: OutputMode): string {
         if (auditResult.comingSoon === true)
           return;
 
-        let lineItem = ` ${log.doubleLightHorizontal} ${formatAggregationResultItem(auditResult.score)} ${auditResult.description}`;
+        let lineItem = ` ${log.doubleLightHorizontal}`;
+        if (auditResult.error) {
```

Comment:
nit: I think keeping the initialization inline and having a 
`const formattedScore = auditResult.error ? log.redify('') : formatAgg...` might be slightly more understandable then the concatenation but not up in arms over in